infrastructure/app/chart/energy-apps/templates/deployment.yaml

```diff
@@ -51,6 +51,8 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           env:
+            - name: APP_VERSION
+              value: {{  .Values.image.tag }}
```

Comment:
I'm unable to get the commit sha into the `values.yaml` or `<stage>.yaml` files.  Neither the `.Values.image.tag` or the `github.sha` methods are available there.

Reply:
out of interest, where is this env var used, is it for the app to knows its own version? I'm a bit confused about what this PR tries to achieve

Reply:
Sorry - I've updated the description.  It follows on from this PR https://github.com/citizensadvice/energy-apps/pull/483